template.haml

```diff
@@ -146,6 +146,7 @@
                 - all_bugs.each do |b|
                   %tr
                     %td
+                      = b.beta_blocker ? "<span class='badge badge-info'>Beta</span>" : ''
```

Comment:
Sudha has requested that the Beta blocker be first in the sort order.  It looks like we currently sort by alphabetical desc to force the blank strings to the end.  Looks like we have two options...

1. The "Right Way" would be to write our own sorting priority that puts things in the order we want and somehow get datatables to use that.
2. Change `Beta` to `ßeta` because the greek alphabet comes after the roman characters when sorting.

@nhr thoughts?

Reply:
I'm confused. You want Beta to show up first in a list that sorts alphabetically. 'B' comes before 'T' for Test and 'O' for Ops & Online...

Reply:
It currently sorts alphabetically *descending*.  So its in reverse order.  Which works because the currently sorting method considers an empty string to come "before" the other strings.  Currently, the blocker columns looks like...

* Test
* Ops
* Online
* Beta
* <Empty string>

So Beta needs to change to make it come after the other test options alphabetically, which will make it first in the list.

Reply:
See http://file.rdu.redhat.com/~rthrashe/shiftzilla/ for it in action.

Reply:
Roger, thanks for reminding me how this code works :-) I like the 'ß' hack.